lib/src/main/java/com/evervault/models/TokenResult.java

```diff
@@ -0,0 +1,6 @@
+package com.evervault.models;
+
+public class TokenResult {
+    public String token;
+    public long expiry;
```

Comment:
Could this be an `Instant`?

Reply:
Won't work unfortunately. Gson can't handle the conversion from MS to an Instant